docs/license.rst

```diff
@@ -0,0 +1,132 @@
+.. _license-page:
```

Comment:
This file has copied most of the text from pretrained models. Why not just update pretrained models page?

Reply:
Good point - I have now simplified the PR and updated `pretrained.rst` instead